padertorch/contrib/examples/pit/evaluate.py

```diff
@@ -193,7 +160,7 @@ def main(_run, batch_size, datasets, debug, experiment_dir):
                 s = s[:, :z.shape[1]]
                 z = z[:, :s.shape[1]]
                 entry['mir_eval'] \
-                    = pb.evaluation.mir_eval_sources(s, z, return_dict=True)
+                    = pb_bss.evaluation.mir_eval_sources(s, z, return_dict=True)
```

Comment:
You should probably use all evaluations possible in pb_bss.evaluation since that is one of the benefits of pb_bss